gitVersionConfig.yaml

```diff
@@ -0,0 +1,2 @@
+assembly-versioning-scheme: MajorMinor
+mode: ContinuousDeployment
```

Comment:
Shouldn't this be `continuous-deployment`?  (https://gitversion.readthedocs.org/en/latest/reference/continuous-deployment/)